install/kubernetes/helm/subcharts/grafana/templates/gateway.yaml

```diff
@@ -0,0 +1,62 @@
+{{- if .Values.gateway.enabled }}
```

Comment:
As mentioned in main comment, an add-on gateway should not be enabled if the add-on itself is not enabled - so needs `.Values.enabled and .Values.gateway.enabled`.

Reply:
I fixed this - all the conditionals now "and" both the enabled and gateway.enabled flags:

```
{{- if and (.Values.enabled) (.Values.gateway.enabled) }}
```